### PR TITLE
Run scripts locally

### PR DESCRIPTION
**Pull request by clayrisser**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -16,10 +16,11 @@
   },
   "scripts": {
     "prepublish": "npm run build",
-    "test": "jest",
-    "lint": "tslint 'src/**/*.{ts,tsx}' --project ./",
+    "test": "node node_modules/jest/bin/jest",
```
Comment (Place1) [inline package.json:19]: what's the reason behind these changes?

Comment (Place1) [reply package.json:19]: and the other script changes etc.

Comment (clayrisser) [reply package.json:19]: I did not realize that npm can figure out local scripts, so actually, this commit isn't needed.
